tools/hsqldb-connection/db-connect.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
you should also create a corresponding .bat script

Reply:
ok done. Could you check the .bat ?